Packages/DependenciesHunter/Editor/DependenciesMapUtilities.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections.Generic;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+
+// ReSharper disable once CheckNamespace
+namespace DependenciesHunter
```

Comment:
I have intentionally made a single file with all the classes knowing it is an antipattern.
Since I assume that a lot of people will just copy-paste it without cloning or using it as a package. I mentioned it in README.
So that was a plan )

Reply:
Yeah, I understood this when used it in my real project :)
It was easy to just copy and paste one file.

"But now, since the package could be distributed with UPM, it's possible to refuse this antipattern" - thought I when ended package setup :)

If you think it would be better to leave one file, then I will fix this

Reply:
Well, I think that would be better to leave one file. So if you fix it that would be great.
That would be DependencyHunter.cs 
with
// ReSharper disable once CheckNamespace. 
comment as before